src/OpenSpaceToolkit/Astrodynamics/NumericalSolver.cpp

```diff
@@ -0,0 +1,407 @@
+        {
+            switch (stepperType_)
+                {
+                    case NumericalSolver::StepperType::RungeKuttaCashKarp54:
+                        integrate_const (make_controlled( absoluteTolerance_, relativeTolerance_, NumericalSolver::error_stepper_type_54() ), aSystemOfEquations, aStateVector, (0.0), propDurationInSecs, adjustedTimeStep, NumericalSolver::NumericalIntegrationLogger) ;
+                        return aStateVector ;
+                    case NumericalSolver::StepperType::RungeKuttaFehlberg78:
+                        integrate_const (make_controlled( absoluteTolerance_, relativeTolerance_, NumericalSolver::error_stepper_type_78() ), aSystemOfEquations, aStateVector, (0.0), propDurationInSecs, adjustedTimeStep, NumericalSolver::NumericalIntegrationLogger) ;
+                        return aStateVector ;
+                }
+        }
+        case NumericalSolver::LogType::LogAdaptive:
+        {
+            switch (stepperType_)
+                {
+                    case NumericalSolver::StepperType::RungeKuttaCashKarp54:
+                        integrate_adaptive (make_controlled( absoluteTolerance_, relativeTolerance_, NumericalSolver::error_stepper_type_54() ), aSystemOfEquations, aStateVector, (0.0), propDurationInSecs, adjustedTimeStep, NumericalSolver::NumericalIntegrationLogger) ;
+                        return aStateVector ;
+                    case NumericalSolver::StepperType::RungeKuttaFehlberg78:
+                        integrate_adaptive (make_controlled( absoluteTolerance_, relativeTolerance_, NumericalSolver::error_stepper_type_78() ), aSystemOfEquations, aStateVector, (0.0), propDurationInSecs, adjustedTimeStep, NumericalSolver::NumericalIntegrationLogger) ;
+                        return aStateVector ;
+                }
+        }
+    }
+
+    throw ostk::core::error::runtime::Undefined("No StateVector returned from Odeint") ;
+    return NumericalSolver::StateVector(anInitialStateVector.size()) ;
+
+}
+
+
+
+String                          NumericalSolver::StringFromLogType          (   const   NumericalSolver::LogType&   aLogType                                    )
+{
+
+    switch (aLogType)
+    {
+
+        case NumericalSolver::LogType::NoLog:
+            return "NoLog" ;
+
+        case NumericalSolver::LogType::LogConstant:
+            return "LogConstant" ;
+
+        case NumericalSolver::LogType::LogAdaptive:
+            return "LogAdaptive" ;
+
+        default:
+            throw ostk::core::error::runtime::Wrong("Integration Stepper Type") ;
+
+    }
+
+    return String::Empty() ;
+
+}
+
+String                          NumericalSolver::StringFromStepperType      (   const   NumericalSolver::StepperType&  aStepperType                             )
+{
+
+    switch (aStepperType)
+    {
+
+        case NumericalSolver::StepperType::RungeKuttaCashKarp54:
+            return "RungeKuttaCashKarp54" ;
+        case NumericalSolver::StepperType::RungeKuttaFehlberg78:
```

Comment:
Let's be consistent with our newlines (+ apply to other `switch` cases in this file):

```suggestion
            return "RungeKuttaCashKarp54" ;

        case NumericalSolver::StepperType::RungeKuttaFehlberg78:
```

Reply:
fixed